app/views/reports/regions/_header.html.erb

```diff
@@ -37,6 +37,11 @@
       <p class="mr-12px mr-lg-24px mb-0px pb-8px fs-16px fs-lg-20px fw-bold <%= params[:action] == "cohort" ? "c-red bb-2px bb-red" : "c-grey-dark" %>">
         <%= link_to_unless_current "Cohort reports", reports_region_cohort_path(@region, report_scope: params[:report_scope]), :class => "c-grey-dark td-none" %>
       </p>
```

Comment:
The other names also need to be prefixed with HTN, for ex `Overview` becomes `HTN Overview`. Should that be part of this PR?

Reply:
We've created a separate story for this.